src/Lecture2.hs

```diff
@@ -198,7 +283,13 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] [] = []
+merge [] xs = xs
```

Comment:
You can remove the first case with two patterns here because it's already covered by the second line 👍🏻